src/components/Sheet/README.md

```diff
@@ -1,20 +1,44 @@
-| visible                  | `boolean`  |    ✓     |             | Show/hide sheet                                                                                                                                                             |
-| allowHideOnContentScroll | `boolean`  |          | `true`      | Enable the behavior in which you can close the sheet window with a swipe down if the content is scrolled to its top (`contentNode.scrollTop === 0`) or has no scroll at all |
-| hideTopBar               | `boolean`  |          |             | Hide top bar with resize handle                                                                                                                                             |
-| id                       | `string`   |          | `modal`     | ID of the sheet, used as hash in URL. It's important to specify different `id` values if there can be more than one sheet on the page                                       |
-| title                    | `string`   |          | `undefined` | Title of the sheet window                                                                                                                                                   |
-| className                | `string`   |          | `undefined` | Class name for the sheet window                                                                                                                                             |
-| contentClassName         | `string`   |          | `undefined` | Class name for the sheet content                                                                                                                                            |
-| swipeAreaClassName       | `string`   |          | `undefined` | Class name for the swipe area                                                                                                                                               |
-| onClose                  | `function` |          | `undefined` | Function called when the sheet is closed (when `visible` sets to `false`)                                                                                                   |
+<!--/GITHUB_BLOCK-->
+
+```tsx
+import {Sheet} from '@gravity-ui/uikit';
+```
+
+`Sheet` is a component designed to be used in a mobile context as an information or interactive element. You can place content of any size in it - internal scrolling and dynamic resizing are supported.
+
+On mobile devices, you can move `Sheet` by pulling on its main part or the swipe area. To close it, swipe down or touch the area outside the `Sheet`.
+
+## Usage
+
+```tsx
+import React from 'react';
+import {Button, Sheet} from '@gravity-ui/uikit';
+
+const [visible, setVisible] = React.useState(false);
```

Comment:
I suggest to wrap this content to function so that code will be usable via copy-paste